setup.py

```diff
@@ -54,4 +54,11 @@
           'tornado',
       ],
       test_suite='test',
+      extras_require={
+          'secure': [
```

Comment:
this means it wont have any effect on 2.7.8 and lower, where it is still needed.